### PR TITLE
delete okeys

### PR DESCRIPTION
**Pull request by pefontana**
I delete the "okeys" and variants from the book, except for chapter one because Herman is working on it

### CODE REVIEW

**Comment by marianLambda** [comment]
a suggestion for commiting: it is good to commit often. As an example, you may want to make a commit each time you finish deleting "okeys" in a chapter and then go to the next one.
Try making a commit every time you finish a "unit" of your work.

File: 12_ultima_continued/12_ultima_continued.jl [hunk 1]
```diff
@@ -134,7 +134,7 @@ And finally we arrive at the day when the madness begins.
...
 
-After the champagne, hugs and a little celebration Garriot return to the work and start to analize the metrics to see if everything is okey, and it does. He relax a little bit until something calls his attention: The curves of carnivorous and herbivorous animals are a little different than they should be. There are still **too few points** (only four hours from the release) to be alarmed, but he decide to do a deeper analysis. Luckily, a few days ago, he had read a paper on the Universal ODEs, so he thinks they can help him in this case.
+After the champagne, hugs and a little celebration Garriot return to the work and start to analize the metrics to see if everything is good, and it does. He relax a little bit until something calls his attention: The curves of carnivorous and herbivorous animals are a little different than they should be. There are still **too few points** (only four hours from the release) to be alarmed, but he decide to do a deeper analysis. Luckily, a few days ago, he had read a paper on the Universal ODEs, so he thinks they can help him in this case.
```
Comment (martinacantaro) [inline 12_ultima_continued/12_ultima_continued.jl:137]: change "good" for "alright"

**Review by martinacantaro — CHANGES_REQUESTED** [review]
The HTML files contain errors, please generate them again

**Comment by pefontana** [comment]
HTML files are corrected and I made the change grammar changes you suggested


Can someone review them?

**Review by marianLambda — APPROVED** [review]

**Review by martinacantaro — APPROVED** [review]

**Pull request merged**